package.json

```diff
@@ -53,10 +52,12 @@
     "eslint-config-prettier": "^8.9.0",
     "eslint-plugin-jest": "^27.2.3",
     "husky": "^8.0.3",
+    "jest": "^29.6.4",
```

Comment:
jest was already installed in the `devDependencies` (because we only need the library to run locally, we don't need it to run in the built/production app). This additional entry in package.json can be removed (make sure to do an `npm install` afterwards to ensure that package-lock.json is synched up).

Reply:
Done commit 2ec36f9fe1a95d8de35f1d20648fb546b6b15b24